### DISCUSSION TITLE
Itemize the principles page to be referred with anchor links

### DISCUSSION THREAD

**Discussion by samueltauil** [open-gitops/project#166] [Ideas] [answered]
I'm going thru the creation of questions on certiverse for the CGOA and something came up, we are referring to glossary topics with very specific anchor links, which gives context for a specific question and also helps community to point to specific topics. I was wondering if it would be good to do the same with the principles page too.

https://github.com/open-gitops/documents/blob/main/PRINCIPLES.md

@chris-short @christianh814

**Comment by christianh814** [comment, marked as answer]
You mean anchor links to each principle?

If so +1 from me

**Reply by samueltauil** [reply]
Yes, I think everyone would benefit from it.
